nbconvert/filters/tests/test_markdown.py

```diff
@@ -175,8 +183,8 @@ def test_markdown2rst(self):
         tokens[1] = r'\*\*test'
 
         for index, test in enumerate(self.tests):
-            self._try_markdown(markdown2rst, test, tokens[index])
-
+            self._try_markdown(partial(convert_pandoc, from_format='markdown',
```

Comment:
Ok, tried to change it to take this into account…do the new warning strings look ok?